### ISSUE TITLE
Issues with dynamically loaded content

### ISSUE THREAD

**Issue by nadchif** [michaeldera/image-accessibility-extension#8]
**Describe the bug**
Right now everything works just well, when Image a11y processes the page on initial load. However, when content is loaded or changes dynamically, the alt labels are not updated. This leads to scenario where 'apples' are still labelled as 'oranges'.

**To Reproduce**
1. Download this gist https://gist.github.com/nadchif/211ecfab904d53880fb8e021c44c7f0b
2. Serve on localhost with Image a11y enabled

**Expected behavior**
Detect image src  and DOM changes in order to update the alt labels dynamically

**Desktop (please complete the following information):**
 - Browser Chrome
 - Version 81

**Comment by michaeldera** [comment]
Interesting observation, what would be the best way to listen on the DOM for dynamic changes to images 🤔... there should be a way to listen to each element as opposed to a diff on all elements right?

**Comment by nadchif** [comment]
A good place to start maybe https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver.

1. On the `body` --> track newly added `img` elements
2. On `img` where we add an `alt`--> track `src` changes

**Comment by michaeldera** [comment]
Yes... that looks like it can work pretty well. So just added a Mutation Observer to each `img` child's  `src` when we are add alt attributes....

We would also need need to add a mutation observer to the whole body and listen for new `img` elements as well.
